### PR TITLE
add /learning-objects/stats route in gateway

### PR DESCRIPTION
**Pull request by shanioffe**
Implement the stats route in gateway, based off of clark-service-refactor

### CODE REVIEW

File: src/modules/clark/metric-module/metrics.routes.ts [hunk 1]
```diff
@@ -14,4 +14,8 @@ export const METRICS_ROUTES: ProxyRoute[] = [
         method: HTTPMethod.GET,
         path: "/users/metrics",
     },
+    {
+        method: HTTPMethod.GET,
+        path: "/learning-objects/stats",
+    },
```
Comment (curlyLasagna) [inline src/modules/clark/metric-module/metrics.routes.ts:20]: when do we need to have `target: envConfig.getUri(CLARK_SERVICE_URI)` in our routes? Or is it always included for any service-refactors?

Comment (johngonz0) [reply src/modules/clark/metric-module/metrics.routes.ts:20]: +1 also curious as to why no target

**Review by johngonz0 — COMMENTED** [review]
Looks good, just luis' comment about the target

File: src/modules/clark/metric-module/metrics.routes.ts (hunk 1 above)
Comment (mfranz1) [reply src/modules/clark/metric-module/metrics.routes.ts:20]: You do need the target here

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Review by Greek — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by curlyLasagna — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Pull request merged**